webpack/assets/javascripts/react_app/components/ChartBox/ChartBoxSelectors.js

```diff
@@ -0,0 +1,14 @@
+import {
+  selectAPIResponse,
+  selectAPIError,
+  selectAPIStatus,
+} from '../../redux/API/APISelectors';
+
+export const selectChart = selectAPIResponse;
+
+export const selectError = (state, key) => {
+  const error = selectAPIError(state, key);
+  return error && error.message;
+};
```

Comment:
you can use `selectAPIErrorMessage` instead

Reply:
Thanks for the suggestion, changed!